app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -53,14 +53,15 @@ class PocketVideoFetchScheduler(
             .setRequiredNetworkType(NetworkType.CONNECTED)
             .build()
 
+        // This sets the custom fetch delay for testing purposes.
+        // The custom delay will only run once.
         val delay: Long
         val workPolicy: ExistingWorkPolicy
 
```

Comment:
nit: since you put the comment above, I'd remove this newline too: basically, the whole block, including the variable declarations, is performing that functionality so, to me, it makes sense to group it all together.